version/version.go

```diff
@@ -51,14 +50,14 @@ func CheckForUpdate(out io.Writer) error {
 	}
 
 	// fetch latest cli version
-	latestTagResp, err := api.RepoLatestRequest("astronomer", "astro-cli")
+	latestTagResp, err := client.RepoLatestRequest("astronomer", "astro-cli")
```

Comment:
I need to figure out how to mock `RepoLatestRequest` and `RepoTagRequest` because the responses should both be different to effectively test all of the cases.

Reply:
Currently, all of the client responses are the same.